src/mono/mono/metadata/debug-mono-ppdb.c

```diff
@@ -96,6 +105,8 @@ get_pe_debug_info (MonoImage *image, guint8 *out_guid, gint32 *out_age, gint32 *
 			if (dir.signature == 0x53445352) {
 				memcpy (out_guid, data + 4, 16);
 				*out_age = read32(data + 20);
+				if (pdb_path)
+					*pdb_path = (char*) data + 24;
```

Comment:
Also here - should we try to find the end of the string and make sure it doesn't go past the end of the debug info